### DISCUSSION TITLE
Malware warning on Windows

### DISCUSSION THREAD

**Discussion by MatthiGamer** [gaphor/gaphor#3157] [General]
If I choose to download the portable version on Windows, Firefox blocks the download and tells me, that the file contains a virus. Virustotal agrees. What part of the application may cause this behavior and why should I trust the application anyways?

**Comment by danyeaw** [comment]
Hi @MatthiGamer we are sorry that you received scary warnings. We could use your help submitting false positive reports to the anti-virus companies:

Firefox uses Google: https://safebrowsing.google.com/safebrowsing/report_error/?hl=en

For convenience, a quick list of shame for the more error-prone vendors is given below. For a more extensive list, see [this page](https://www.techsupportalert.com/how-to-report-malware-or-false-positives-to-multiple-antivirus-vendors/).

* [Microsoft Defender](https://www.microsoft.com/wdsi/filesubmission) (previously called *Windows Defender*)
* [MacAfee](https://www.microsoft.com/en-us/wdsi/filesubmission) (requires a paid *corporate* account to submit files)
* [AVG](https://www.avg.com/en-us/false-positive-file-form)
* [Qihoo-360](http://www.360totalsecurity.com/en/suspicion.html)
* [Zillya](mailto:antivirus@zillya.com?subject=False%20Positive%20Submission&body=The%20sample%20is%20in%20a%20password%20protected%20zip%20file%0A%0AThe%20password%20for%20the%20attachment%20is%20infected) (via email only)

This is most likely cause by us bundling our app using PyInstaller. You should trust the application because it is all open source and transparent. You can see in the GitHub Action definitions how the portable installer is built.
